RxSugar/ObserverBinding.swift

```diff
@@ -17,7 +17,7 @@ precedence 141
  - parameter observable: Observable that sends events.
  - returns: Disposable object that can be used to unsubscribe the observer.
  */
-@warn_unused_result(message="http://git.io/rxs.ud")
```

Comment:
I would keep this since this is probably a programmer error if they forget to actually use the disposable.